specs/disk_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,111 @@
+= Specification for adding disk details in node hardware inventory
+
+Add disk details also as a part of node hardware inventory
+
+== Problem description
+
+Currently disk details are missing in node hardware inventory. Add disk details
+in node hardware inventory and disk details should contain device_name, fs_type,
+fs_uuid, model, mount_point, parent, size, name, disk_type, used, ssd, vendor,
+disk_id, node_id.
+
+== Use Cases
+
+Tendrl should show the current status of disk.
+
+== Proposed change
+
+* Modify manager module to find disk details and update those details into
+  node_inventory list.
+* Create a new class in persistence module to keep disk details.
+* Populate the object using node_inventory list and persist the object in etcd.
+
+== Alternatives
+
+* Util-linux package is used to find disk details of particular node.
+* Add Util-linux as dependency.
+
+== Data model impact
+
+Add disk details in etcd with key nodes/node_id/Disks/device_name
+
+== Impacted modules
+
+== Tendrl API impact:
+
+Api to get disk details v2/keys/nodes/node_id/Disks
+
+=== Tendrl/common impact
+
+None
+
+=== Tendrl/node_agent impact
+
+Modify persistence and manager modules to find and store disks details.
+
+=== Tendrl/gluster_integration impact
+
+None
+
+=== Tendrl/ceph_integration impact
+
+None
+
+
+== Security impact
+
+None
+
+== Notifications/Monitoring impact
+
+We might have to monitor the status of the disk.
```

Comment:
How? If you think this should be addressed as part of another spec, raise github issues and link it here